server/command/install.go

```diff
@@ -16,9 +16,11 @@ import (
 func (s *service) executeInstall(params *params) (*model.CommandResponse, error) {
 	appID := ""
 	appSecret := ""
+	appWebhookSecret := ""
 	fs := pflag.NewFlagSet("", pflag.ContinueOnError)
 	fs.StringVar(&appID, "app-id", "", "ID of the app")
 	fs.StringVar(&appSecret, "app-secret", "", "App secret")
+	fs.StringVar(&appWebhookSecret, "app-webhook-secret", "", "App webhook secret")
```

Comment:
do we have to use a secret different from the app secret? (or can we reuse it if not provided in the command)?

Reply:
Good question.  We could use the same secret.  I'm not sure if he have used that implementation in another plugin

Reply:
As @hanzei pointed out, AppSecret is specific to the HTTP apps. I like @larkox's suggestion of a generated secret, but it's more code; I'm ok with a manually entered, separate secret for now.